templates/partner/intel.html

```diff
@@ -0,0 +1,645 @@
+                                hi_def=True,
+                                attrs={"class": "p-image-container__image"},
+                                loading="lazy") | safe
+                }}
+              </div>
+            </div>
+            <div class="p-equal-height-row__item">
+              <hr class="p-rule--highlight u-hide--medium u-hide--small" />
+              <h3 class="p-heading--5">Telco</h3>
+            </div>
+            <p class="p-equal-height-row__item">
+              Benefit from carrier-grade solutions based on open source for core, regional and edge telco clouds, including OpenStack, Kubernetes and Ubuntu for Intel. Reduce infrastructure costs whilst maintaining reliable, secure and feature-rich networks.
+            </p>
+            <div class="p-equal-height-row__item">
+              <hr class="p-rule--muted u-hide--small u-hide--medium" />
+              <p>
+                <a href="/telco">Learn more about Telco use cases&nbsp;&rsaquo;</a>
+              </p>
+            </div>
+          </div>
+          <div class="p-equal-height-row__col">
+            <div class="p-equal-height-row__item">
+              <div class="p-image-container--3-2-on-small p-image-container--square-on-medium p-image-container--2-3-on-large is-highlighted">
+                {{ image(url="https://assets.ubuntu.com/v1/d24fcc48-industrial.png",
+                                alt="",
+                                width="852",
+                                height="1279",
+                                hi_def=True,
+                                attrs={"class": "p-image-container__image"},
+                                loading="lazy") | safe
+                }}
+              </div>
+            </div>
+            <div class="p-equal-height-row__item">
+              <hr class="p-rule--highlight u-hide--medium u-hide--small" />
+              <h3 class="p-heading--5">Industrial</h3>
+            </div>
+            <p class="p-equal-height-row__item">
+              Intel and Canonical have partnered to ensure easy access to advanced silicon for embedded solutions, with the immutable Ubuntu Core operating system - our embedded OS that is lightweight and tailored to run on specific Intel hardware.
+            </p>
+            <div class="p-equal-height-row__item">
+              <hr class="p-rule--muted u-hide--small u-hide--medium" />
+              <p>
+                <a href="/industrial">Learn more about Industrial use cases&nbsp;&rsaquo;</a>
+              </p>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <hr class="p-rule is-fixed-width" />
+
+  <section class="p-strip">
+    <div class="u-fixed-width">
+      <h2>
+        <a href="/ai/mlops">Learn how to streamline your AI workflow with MLOps&nbsp;&rsaquo;</a>
+      </h2>
+    </div>
+  </section>
+
+  <section class="p-section">
+    <div class="p-section--shallow">
```

Comment:
Can you remove the shallow padding on `p` and apply it to the row instead? Currently the shallow padding is also applied on the image, which should not be the case. The image container wrapper will add shallow padding to the image.

Reply:
![image](https://github.com/user-attachments/assets/2d2b90e1-d8c3-4097-8208-a7100b9ccf54)
It does not but I can apply it to row